awx/api/serializers.py

```diff
@@ -1697,6 +1697,7 @@ def get_summary_fields(self, obj):
         d.setdefault('recent_jobs', [{
             'id': j.job.id,
             'name': j.job.job_template.name if j.job.job_template is not None else "",
+            'type': j.job.get_real_instance_class()._meta.verbose_name.replace(' ', '_'),
```

Comment:
We do this here as well: https://github.com/ansible/awx/blob/devel/awx/main/models/unified_jobs.py#L1224

It might be worth it to just make this a property on the UJ model & add simple unit test to check each job type against a list of expected values.

I could also see it being too early to tell and wanting to hold off. Up to you.

Reply:
Done!